### PR TITLE
Assignment: Hello Core Logic

### PR DESCRIPTION
**Pull request by wadesun18**

### CODE REVIEW

File: assignment_1/hello.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ function displayBoard(boardName){
     var map = new Map(Object.entries(hello[boardName]));
     map.forEach(function(item,index){
         console.log("|------------------");
-        console.log("| " +index)
+        console.log("| " + index)
         console.log("|------------------");
           for (var i=0; i<item.length; i++){
```
Comment (williamtstanley) [inline assignment_1/hello.js]: space seperate operators `(var i = 0; i < item.length; i++)`

**Comment by williamtstanley** [comment]
Space your operators etc. The code becomes easier to read that way.
Also consider if there are ways you reduce the nested if statements. Some of them could be && for example.

**Comment by wadesun18** [comment]
Sounds good, thanks Travis!

**Comment by williamtstanley** [comment]
Object.entries isnt correct. Currently broken.

**Comment by williamtstanley** [comment]
also why new Map?

**Pull request merged**